inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -53,28 +54,51 @@ class StaticShapeNMS final : public NonMaxSuppression {
 }  // namespace
 
 void FrontEnd::parseStaticShapeNMS(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const {
-    VPU_THROW_UNLESS(inputs.size() >= 2 && inputs.size() <= 5,
-        "StaticShapeNMS parsing failed, expected number of input is in range [2, 5], but {} provided",
-        inputs.size());
-    VPU_THROW_UNLESS(outputs.size() == 2,
-        "StaticShapeNMS parsing failed, expected number of outputs: 2, but {} provided",
-        outputs.size());
+    VPU_THROW_UNLESS(inputs.size() == 6,
+        "StaticShapeNMS with name {} parsing failed, expected number of inputs: 6, but {} provided",
+        layer->name, inputs.size());
+    VPU_THROW_UNLESS(outputs.size() == 4,
+        "StaticShapeNMS with name {} parsing failed, expected number of outputs: 4, but {} provided",
+        layer->name, outputs.size());
+
+    const auto softNMSSigmaData = inputs[5];
+    VPU_THROW_UNLESS(softNMSSigmaData->usage() == DataUsage::Const,
+        "StaticShapeNMS with name {} parsing failed: softNMSSigma should have usage {} while it actually has {}",
+        layer->type, DataUsage::Const, softNMSSigmaData->usage());
+    VPU_THROW_UNLESS(softNMSSigmaData->desc().totalDimSize() == 1,
+        "StaticShapeNMS with name {} parsing failed: softNMSSigma input should contain 1 value, while it has {} values",
+        layer->type, softNMSSigmaData->desc().totalDimSize());
+    const auto softNMSSigma = InferenceEngine::PrecisionUtils::f16tof32(softNMSSigmaData->content()->get<InferenceEngine::ie_fp16>()[0]);
+    VPU_THROW_UNLESS(softNMSSigma == 0,
+        "StaticShapeNMS with name {} parsing failed: the only supported value for softNMSSigma is 0, while it actually equal to  {}",
```

Comment:
Nitpick: extra space before the last `{}` in error message

Reply:
Good catch! Fixed.